src/js/Setup.js

```diff
@@ -256,8 +256,8 @@ function createCindyNow() {
 }
 
 function loadExtraModules() {
```

Comment:
On 03/20/2016 12:35 PM, nasay wrote:
>
> In src/js/Setup.js
> <https://github.com/CindyJS/CindyJS/pull/304#discussion_r56762833>:
>
> > @@ -256,8 +256,8 @@ function createCindyNow() {
> >  }
> >  
> >  function loadExtraModules() {
>
> I am not sure: are the functions |loadExtraModules| and
> |loadExtraPlugin| still needed?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/CindyJS/CindyJS/pull/304/files/e1e62973386fd0bd9ab761a199df911deb96d0f5#r56762833>
>
Hi,

for convexhull3d the functions will be unnecessary as soon we replaced
the plugin, nevertheless i'm planning to use them for CSV support
(https://github.com/strobelm/CindyJS/blob/csv/src/js/Setup.js#L262),
therefor removing them will cause merge conflicts later on (in the case
my branch will get merged).

-- 
Michael Strobel, M.Sc.

Technische Universität München -- Zentrum Mathematik
Lehrstuhl für Geometrie und Visualisierung (M10)
Boltzmannstr. 3, 85747 Garching bei München 
https://www-m10.ma.tum.de/bin/view/Lehrstuhl/MichaelStrobel